src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+// variables for header
+$black: #060b35;
+$blue: #00acdc;
```

Comment:
Avoid variable names by their meaning, they should indicate and that's exactly what we use them for.